src/components/TextField/TextFieldEnums.js

```diff
@@ -0,0 +1,11 @@
+// TextFieldVariant enum used by the TextField variant prop
+export const TextFieldVariant = Object.freeze({
+    DEFAULT: 'default',
+    TITLE: 'title',
+});
+
+// TextAreaAutoHeight enum used by the TextArea autoHeight prop
+export const TextAreaAutoHeight = Object.freeze({
+    ENABLED: true,
+    DISABLED: false,
+});
```

Comment:
As per my comment in the TextArea, I think we don't need this enum, we should just use a boolean.

Reply:
Sorted